README.md

```diff
@@ -50,10 +50,22 @@ To build it, run:
 
     mvn -DskipTests clean package
 
+Build against Spark 3.2.x
+
+    mvn -DskipTests clean package -Pspark3.2
+
+Build against Spark 3.2.x and Spark version
+
+    mvn -DskipTests clean package -Pspark3.2 -Dspark.version=3.2.1
+
 To package the Firestorm, run:
 
     ./build_distribution.sh
 
+Package against Spark 3.2.x, run:
+
+    ./build_distribution.sh --spark3-mvn "-Pspark3.2"
```

Comment:
Thank you for your information.
But we don't seem to have a way to build a specified profile now.
If you use `./build_distribution.sh -Pspark3.2 -Dspark.version=3.2.0`, Spark2 compilation will fail because mvn options are global.

Reply:
We should have a way to pass the parameter profile. Parameter `profile` is ok.But we shouldn't teach users to  pass the parameter `spark.version` in the document. Could we remove  the use of `spark.version` in the document?

Reply:
OK